Summers_James/detail.html

```diff
@@ -0,0 +1,32 @@
+<%inherit file="./catalog/templates/app_base.htm" />
+
+
+
+<%block name="content">
+<div class="persCata">
+    <h1 style="left:45%; position: relative; top:18px;">${product.name}</h1>
```

Comment:
Remove in-line styling and put in app_base.css
Do same with other in-line styling in this file.

Reply:
+1